openbb_terminal/common/behavioural_analysis/twitter_model.py

```diff
@@ -172,7 +172,11 @@ def get_sentiment(
             end_date=dt_recent.strftime(dt_format),
         )
 
-        if temp.empty:
+        # given object is a dataframe and is empty
+        if isinstance(df_tweets, pd.DataFrame) and df_tweets.empty:
+            return pd.DataFrame()
+        # given object is not a dataframe and is None
+        if not isinstance(df_tweets, pd.DataFrame) and not df_tweets:
```

Comment:
Why not do:
if not df_tweets:
        return pd.DataFrame()
if df_tweets.empty:
    return pd.DataFrame()
(sorry its terribly formatted)